frontend/src/modules/individualDetails/reducers.ts

```diff
@@ -0,0 +1,102 @@
+import { WritableDraft } from "immer/dist/types/types-external";
+import { createNetworkActionTypes } from "../../utils/actions";
+import { AnyAction } from "redux";
+import { FetchedState } from "../common";
+import { FMSIndividual, FMSSample } from "../../models/fms_api_models";
+import produce from "immer";
+import { PagedItems } from "../../models/paged_items";
+import { createItemsByID } from "../../models/frontend_models";
+import { clearFiltersReducer, setFilterReducer } from "../../components/shared/WorkflowSamplesTable/FilterReducers";
+
+export const LIST_TABLE = createNetworkActionTypes('INDIVIDUAL_DETAILS.LIST_TABLE')
+export const SET_SORT_BY = 'INDIVIDUAL_DETAILS.SET_SORT_BY'
+export const SET_INDIVIDUAL_DETAILS_SAMPLES_FILTER = 'INDIVIDUAL_DETAILS.SET_INDIVIDUAL_SAMPLES_FILTER'
+export const CLEAR_FILTERS = "INDIVIDUAL_DETAILS.CLEAR_FILTERS"
+
+export interface Individual {
+    individual: FMSIndividual,
+    samplesByIndividual: PagedItems<FMSSample>
+}
+export interface IndividualDetailsState {
+    [key: number]: Readonly<FetchedState<Individual>>
```

Comment:
Whenever I define a key map like this I add a comment to indicate what is expected as a key value (eg. an individual id, sample id, etc). Also, is the key an FMSId?